libphal/phal_dump.C

```diff
@@ -132,7 +132,17 @@ struct pdbg_target* getTargetFromFailingId(const uint32_t failingUnit,
 
 	pdbg_for_each_class_target(chipTypeString.c_str(), target)
 	{
-		auto targetIdx = pdbg_target_index(target);
+		uint8_t targetIdx = 0;
+		if (!pdbg_target_get_attribute(target,
+							"ATTR_INVENTORY_INDEX",
+							1,
```

Comment:
Rather than passing `1` directly if we can name it to some meaningful variable then would be easier in future to read and understand

Reply:
Adding two variables does not seem like a good idea. The method is invoked everywhere as this. If you still think it is a good idea, I can do it